### PR TITLE
Home

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Review by aayushjain-code — APPROVED** [review]
Try to align the images in our products section to the center. Else it looks great!

**Review by Developers-codz — APPROVED** [review]
UI Looks great. Good Work !

**Comment by MuskanChhatrasal** [comment]
Made the changes. You can view now.

**Comment by MuskanChhatrasal** [comment]
Thankyou!!

**Comment by anik31** [comment]
Center align the view all products button

File: index.html [hunk 1]
```diff
@@ -1,66 +1,228 @@
...
+                    <h3>tasty pizza</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda inventore neque amet ipsa tenetur voluptates
+                        aperiam tempore libero labore aut.</p>
+                </div>
+            </div>
+
+            <div class="box">
+                <img class="image" src="image/s-img-3.jpg" alt="ice-cream">
+                <div class="content">
+                    <img src="image/s-3.png" alt="">
```
Comment (anik31) [inline index.html]: add alt in images

**Comment by anik31** [comment]
There's a small arrow below cold ice cream card, I think it's by mistake.
image

**Comment by MuskanChhatrasal** [comment]
Yes It's by mistake. Will correct that

File: index.html (hunk 1 above)
Comment (MuskanChhatrasal) [reply index.html]: Done.

**Comment by MuskanChhatrasal** [comment]
Done. You can view now.

**Comment by MuskanChhatrasal** [comment]
Removed arrow.

**Comment by anik31** [comment]
> Center align the view all products button

I think this is still there. You can work on it later

**Review by anik31 — APPROVED** [review]
Nice work @MuskanChhatrasal . UI is really good.

**Comment by MuskanChhatrasal** [comment]
Okayy. Will do that.

**Comment by MuskanChhatrasal** [comment]
Thankyou!!

**Pull request merged**